core/src/test/java/org/apache/iceberg/view/TestViewMetadataParser.java

```diff
@@ -335,7 +335,7 @@ public void metadataCompression(String fileName) throws IOException {
             .schemaId(0)
             .versionId(1)
             .timestampMillis(23L)
-            .putSummary("operation", "create")
+            .putSummary("user", "some-user")
```

Comment:
It's probably best to use a fake property here rather than making someone think that `user` is a defined property or would matter. This is very minor though.